stat/statmat_test.go

```diff
@@ -287,7 +289,8 @@ func TestMahalanobis(t *testing.T) {
 				[]float64{
 					0.8, 0.3, 0.1,
 					0.3, 0.7, -0.1,
-					0.1, -0.1, 7}),
+					0.1, -0.1, 7,
+				}),
```

Comment:
Let's leave these formatting changes alone. Please revert this and the changes above in this file.